src/Packages/UnityWebBrowser/Runtime/Core/RawImageUwbClientInputHandler.cs

```diff
@@ -23,7 +23,16 @@ public abstract class RawImageUwbClientInputHandler : RawImageUwbClientManager,
         /// </summary>
         [Tooltip("The input handler to use")]
         public WebBrowserInputHandler inputHandler;
-        
+
+        [Tooltip("Disable the usage of mouse and keyboard")]
+        public bool disableAllControls;
```

Comment:
I think having a global `disableAllControls` control is a bit redundant, as if someone does want to disable all controls, they can just set both `disableMouse` and `disableKeyboard` at once.